packages/contentful--app-scripts/README.md

```diff
@@ -158,22 +158,65 @@ When passing the `--ci` argument adding all variables as arguments is required
 | `--definition-id`        | The ID of the app to which to add the bundle                                                                                                   |
 | `--token`                | A personal [access token](https://www.contentful.com/developers/docs/references/content-management-api/#/reference/personal-access-tokens)     |
 
-**Note:** You can also pass all arguments in interactive mode to skip being asked for it. 
+**Note:** You can also pass all arguments in interactive mode to skip being asked for it.
 
 ### Open Settings of an AppDefinition
 
-It opens the settings in the contentful web app so that you can use the UI to change the settings of an [AppDefinition](https://www.contentful.com/developers/docs/extensibility/app-framework/app-definition/). 
+It opens the settings in the contentful web app so that you can use the UI to change the settings of an [AppDefinition](https://www.contentful.com/developers/docs/extensibility/app-framework/app-definition/).
 
 > **Example**
 >
 > ```shell
 > $ npx --no-install @contentful/app-scripts open-settings --definition-id some-definition-id
 > ```
 
-You can also execute this command without the argument if the environment variable (`CONTENTFUL_APP_DEF_ID`) has been set. 
+You can also execute this command without the argument if the environment variable (`CONTENTFUL_APP_DEF_ID`) has been set.
 
 > **Example**
 >
 > ```shell
 > $ CONTENTFUL_APP_DEF_ID=some-definition-id npx --no-install @contentful/app-scripts open-settings
 > ```
+
+
+### Clean up bundles
+
+Allows you to clean the list of previous bundles. It fetches the list and deletes all bundles except the 50 newest ones.
+You can adjust the amount you want to keep by passing `--keep <amount>` to the argument, if not passed, the default is 50.
+
+#### Interactive mode:
+
+In the interactive mode, the CLI will ask for all required options
+
+> **Example**
+>
+> ```shell
+> $ npx --no-install @contentful/app-scripts bundle-cleanup
+> ```
+
+You can also execute this command without the argument if the environment variable (`CONTENTFUL_APP_DEF_ID`) has been set. 
```

Comment:
Isn't ORG_ID also required?

Reply:
> I like the idea, but hits may end up being flaky. It'll be more stable if we used fake timers (for example https://sinonjs.org/releases/latest/fake-timers/ )

Yep, you are right 👍  I added fake timers to the timeout